config/manager/manager.yaml

```diff
@@ -103,4 +103,4 @@ spec:
       volumes:
         - name: config-volume
           configMap:
-            name: numaplane-controller-config
+            name: controller-config
```

Comment:
Having the prefix made the configmap name `numaplane-numaplane-controller-config`.